### ISSUE TITLE
web.de SmtpError io: incomplete

### ISSUE THREAD

**Issue by jbspeakr** [reacherhq/check-if-email-exists#1109]
### Email Provider

web.de

### What happened?

According to the docs `It might happen on rare occasions that the email provider doesn't allow real-time verification of emails. In this case, there's unfortunately nothing Reacher can do. Please let us know if this happens, we're working on finding ways to solve these issues, which in most occasions are solved on a case-by-case basis.`

Hence, I would like to let you know that there seems to be an unexpected issue when talking smtp with web.de:

```
"smtp": {
    "error": {
      "type": "SmtpError",
      "message": "io: incomplete"
    }
  },
```

**Comment by jbspeakr** [comment]
same for gmx.de

**Comment by amaurym** [comment]
Hey, thanks! I believe the `io: incomplete` errors have been partially solved in v0.8.32. I just tried `***@gmx.de` and `***@web.de`, and they both work well.

```bash
cargo run someone@gmx.de --smtp-port 25 --hello-name reacher.email --from-email amaury@reacher.email
cargo run someone@web.de --smtp-port 25 --hello-name reacher.email --from-email amaury@reacher.email
```
